### PR TITLE
Design static pages

### PR DESCRIPTION
**Pull request by suuhv**
[o] rails_best_practices: success
[o] brakeman: success
[o] bundle-audit: success
[o] rspec: success
[o] rubocop: success
[x] eslint: failed
[o] scss-lint: success
[o] reek: success

### CODE REVIEW

**Comment by suuhv** [comment]
ready

File: app/views/layouts/_header.html.erb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          <ul class="nav navbar-nav navbar-right">
+            <li>
+              <%= link_to t(".signup"), "#",
+                class: "btn btn-default btn-outline btn-circle collapsed" %>
+            </li>
+            <li>
+              <%= link_to t(".signin"), "#",
+                class: "btn btn-default btn-outline btn-circle collapsed" %>
+            </li>
+          </ul>
```
Comment (huongnguyenmta) [inline app/views/layouts/_header.html.erb]: fix all indent from line 8-61

File: app/views/static_pages/home.html.erb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+        <%= f.label :remember_me, class: "checkbox inline" do %>
+          <%= f.check_box :remember_me %>
+          <span><%= t ".remember_me" %></span>
+        <% end %>
+
+        <%= f.submit t(".login"), class: "btn btn-primary" %>
+      <% end %>
+    </div>
+  </div>
```
Comment (huongnguyenmta) [inline app/views/static_pages/home.html.erb]: fix all indent from line 3-27

File: config/routes.rb [hunk 3]
```diff
@@ -1,3 +1,8 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root "static_pages#home"
+  get "/about", to: "static_pages#about"
+  get "/contact", to: "static_pages#contact"
+  get "/home", to: "static_pages#home"
```
Comment (huongnguyenmta) [inline config/routes.rb]: cái này cần nữa không nhỉ?

**Comment by huongnguyenmta** [comment]
commented

File: config/routes.rb (hunk 3 above)
Comment (suuhv) [inline config/routes.rb]: vâng

Comment (suuhv) [inline config/routes.rb]: e xóa luôn ạ (fank)

**Comment by suuhv** [comment]
ready

**Pull request closed without merging**